charts/zero-notifcation-service/values.yaml

```diff
@@ -10,6 +10,11 @@ imagePullSecrets: []
 nameOverride: ""
 fullnameOverride: ""
 
+externalSecret:
+  enabled: true
+  backendType: secretsManager
+  dataFrom: []
```

Comment:
Maybe a comment indicating that this means a list of the secretmanager secret names that data will be pulled from? And maybe even a link to externalsecrets itself for this section?